aptos-move/aptos-vm/src/zkid_validation.rs

```diff
@@ -0,0 +1,145 @@
+// Copyright © Aptos Foundation
+// Parts of the project are originally copyright © Meta Platforms, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::move_vm_ext::{AptosMoveResolver, SessionExt};
+use anyhow::{Context, Result};
```

Comment:
Please please please do not use anyhow or at least use anyhow::Result so it is clearly seen? Otherwise very easy to misuse. In the VM you should use `PartialVMError` or `VMStatus`. Left a comment below.

Reply:
used anyhow::Result